lib/services/local/presence.go

```diff
@@ -1480,6 +1484,74 @@ func (s *PresenceService) DeleteAllWindowsDesktopServices(ctx context.Context) e
 	return s.DeleteRange(ctx, startKey, backend.RangeEnd(startKey))
 }
 
+// ListResources returns a paginated list of resources.
+func (s *PresenceService) ListResources(ctx context.Context, req proto.ListResourcesRequest) ([]types.Resource, string, error) {
+	limit := int(req.Limit)
+
+	var keyPrefix []string
+	var unmarshalItemFunc backendItemToResourceFunc
+
+	switch req.ResourceType {
+	case proto.ResourceType_RESOURCE_TYPE_DATABASE_SERVER:
+		keyPrefix = []string{dbServersPrefix, req.Namespace}
+		unmarshalItemFunc = backendItemToDatabaseServer
+	case proto.ResourceType_RESOURCE_TYPE_APPLICATION_SERVER:
+		keyPrefix = []string{appServersPrefix, req.Namespace}
+		unmarshalItemFunc = backendItemToApplicationServer
+	default:
+		return nil, "", trace.Errorf("%s not implemented at ListResources", req.ResourceType)
+	}
+
+	rangeStart := backend.Key(append(keyPrefix, req.StartKey)...)
```

Comment:
```suggestion
	rangeStart := backend.Key(keyPrefix, req.StartKey)
```

Reply:
That won't work since the `keyPrefix` is a `[]string`, so we need to append the `req.StartKey` and pass them to the `Key` function.